AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -47,6 +47,10 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignItems) {
   ASStackLayoutAlignItemsCenter,
   /** Expand children to fill cross axis */
   ASStackLayoutAlignItemsStretch,
+  /** Children align to their first baseline. Only available for horizontal stack nodes */
+  ASStackLayoutAlignItemsBaselineFirst,
```

Comment:
Yes, I second this. Simplifying these things to AlignItemsBaseline will probably make the API a bit easier to learn. Also, the implementation of ASStackPositionedLayout might be more generic and a bit simpler.

What I'm thinking about is this:
- Instead of ascender and descender properties, ASLayoutable would have a baselineDistance. If it holds a positive value, it equals to ascender. A negative value means descender. (Or maybe the other way around, depends on which way you think is easier to understand)
- Similar to ascender and descender, baselineDistance is 0 for all objects excepts ASTextNode.
- ASTextNode has ascender, descender and a first/last alignment flag. Its baselineDistance can be computed from these properties.
- ASStackPositionedLayout can simply ask for baselineDistance of each item and position it accordingly.

There must be some problems that I overlooked, so please take this comment with a grain of salt :)